src/main/scala/Reader.scala

```diff
@@ -36,12 +36,30 @@ case object SpecFormatYAML extends SpecFormat
 
 class Reader{
   def readFile (fileName:String) = {
-    val content = Source.fromURL(getClass.getResource(fileName)).getLines().mkString("\n")
+    Source.fromURL(getClass.getResource(fileName)).getLines().mkString("\n")
+  }
+
+  def readWebUrl (WbUrl:String) = {
+    Source.fromURL(WbUrl).mkString
+  }
+
+  def specType (fileName:String) = {
```

Comment:
Add a specific return type of Option[SpecFormat] to this method and rename it to specFormat